megamek/src/megamek/client/ui/swing/ForceGenerationOptionsPanel.java

```diff
@@ -370,15 +354,15 @@ public void updateSubfactionChoice() {
         cbSubfaction.removeAllItems();
         FactionRecord selectedFaction = (FactionRecord)cbFaction.getSelectedItem();
         if (selectedFaction != null) {
-            ArrayList<FactionRecord> recs = new ArrayList<>();
+            List<FactionRecord> recs = new ArrayList<>();
             for (FactionRecord fRec : RATGenerator.getInstance().getFactionList()) {
                 if (fRec.getKey().startsWith(selectedFaction.getKey() + ".")
                         && fRec.isActiveInYear(ratGenYear)) {
                     recs.add(fRec);
                 }
             }
-            Collections.sort(recs, factionSorter);
-            cbSubfaction.addItem(null); //No specific subcommand.
+            recs.sort(factionSorter);
+            cbSubfaction.addItem(null); //No specific sub-command.
```

Comment:
Unnecessary hyphen. Subcommand is a perfectly valid spelling.

Reply:
Reverted, and fixed my customized autocorrect for this one.